### PR TITLE
Update team page

### PR DESCRIPTION
**Pull request by sauhardar**
Create cutout to make profile pictures look circular.

### CODE REVIEW

File: src/components/teamPhotos.js [hunk 1]
```diff
@@ -61,11 +61,24 @@ const ProfileRole = styled.span`
 const ProfileImg = styled(Img)`
   picture img {
     transition: filter ease-in 200ms !important;
+    /* border-radius: 50%; */
```
Comment (dajinchu) [inline src/components/teamPhotos.js]: delete this line

File: src/components/teamPhotos.js [hunk 2]
```diff
@@ -143,6 +156,7 @@ const Profile = ({ member, percent }) => {
       <ProfileImgWrapper>
         <ProfileImg
           style={{ overflow: "visible" }}
+          // loading="eager"
```
Comment (dajinchu) [inline src/components/teamPhotos.js]: delete this

File: src/components/teamPhotos.js [hunk 3]
```diff
@@ -61,11 +61,24 @@ const ProfileRole = styled.span`
...
     transition: filter ease-in 200ms !important;
+    /* border-radius: 50%; */
   }
 `
 
 const ProfileImgWrapper = styled.div`
   position: relative;
+  overflow: hidden;
+
+  &::after {
```
Comment (dajinchu) [inline src/components/teamPhotos.js]: I'm curious what the purpose of this is?

File: src/content/team/team.json [hunk 4]
```diff
@@ -10,32 +19,104 @@
...
+      "role": "Team Lead - PharmD Tracker",
+      "profileImage": "./profileImages/tal.png",
+      "socialMedia": {
+        "email": "usvyatsky.t@husky.neu.edu",
+        "linkedIn": "https://www.linkedin.com/in/tal-usvyatsky/"
+      }
+    },
+    {
+      "name": "Jacob Chvatal",
+      "role": "Team Lead - Eliza Project",
```
Comment (dajinchu) [inline src/content/team/team.json]:
```suggestion
      "role": "Team Lead - ELISA Project",
```

File: src/content/team/team.json [hunk 5]
```diff
@@ -10,32 +19,104 @@
...
+      "role": "Head of Brand",
+      "profileImage": "./profileImages/catherine.png",
+      "socialMedia": {
+        "email": "titcomb.c@husky.neu.edu",
+        "linkedIn": "https://www.linkedin.com/in/catherinetitcomb/"
+      }
+    },
     {
       "name": "Fiona Gridley",
       "role": "Design Lead",
```
Comment (dajinchu) [inline src/content/team/team.json]:
```suggestion
      "role": "Designer",
```
I don't think we ever made a Design Lead role

File: src/content/team/team.json [hunk 6]
```diff
@@ -64,66 +145,192 @@
...
+      "name": "Iman Moreira",
+      "role": "Developer",
+      "profileImage": "./profileImages/iman.png",
+      "socialMedia": {
+        "email": "",
+        "linkedIn": ""
+      }
+    },
+    {
+      "name": "Jose Savaria",
```
Comment (dajinchu) [inline src/content/team/team.json]: Move Jose up with the rest of the designers

File: src/content/team/team.json [hunk 7]
```diff
@@ -64,66 +145,192 @@
...
+        "email": "huang.zij@husky.neu.edu",
+        "linkedIn": "https://www.linkedin.com/in/lizzij/"
+      }
+    },
+    {
+      "name": "Iman Moreira",
+      "role": "Developer",
+      "profileImage": "./profileImages/iman.png",
+      "socialMedia": {
+        "email": "",
```
Comment (dajinchu) [inline src/content/team/team.json]: Is this intentionally left blank?

File: src/components/teamPhotos.js [hunk 8]
```diff
@@ -128,28 +128,32 @@ const IconWrapper = styled.div`
 `
 
 const Profile = ({ member, percent }) => {
+  console.log("member")
```
Comment (ArunJ98) [inline src/components/teamPhotos.js]: take these out

**Pull request merged**